lua/plugins/bufferline.lua

```diff
@@ -3,103 +3,103 @@ local colors = require(global_theme)
-        },
-        buffer_selected = {
-            guifg = colors.white,
-            guibg = colors.black,
-            gui = "bold"
-        },
-        -- tabs
-        tab = {
-            guifg = colors.light_grey,
-            guibg = colors.one_bg3
-        },
-        tab_selected = {
-            guifg = colors.black2,
-            guibg = colors.nord_blue
-        },
-        tab_close = {
-            guifg = colors.red,
-            guibg = colors.black
-        },
-        indicator_selected = {
-            guifg = colors.black,
-            guibg = colors.black
-        },
-        -- separators
-        separator = {
-            guifg = colors.black2,
-            guibg = colors.black2
-        },
-        separator_visible = {
-            guifg = colors.black2,
-            guibg = colors.black2
-        },
-        separator_selected = {
-            guifg = colors.black2,
-            guibg = colors.black2
-        },
-        -- modified
-        modified = {
-            guifg = colors.red,
-            guibg = colors.black2
-        },
-        modified_visible = {
-            guifg = colors.red,
-            guibg = colors.black2
-        },
-        modified_selected = {
-            guifg = colors.green,
-            guibg = colors.black
-        },
-        -- close buttons
+bufferline.setup({
+	options = {
+		offsets = { { filetype = "NvimTree", text = "", padding = 1 } },
+		buffer_close_icon = "",
+		modified_icon = "",
+		close_icon = "",
+		left_trunc_marker = "",
+		right_trunc_marker = "",
+		max_name_length = 14,
+		max_prefix_length = 13,
+		tab_size = 20,
+		show_tab_indicators = true,
+		enforce_regular_tabs = false,
+		view = "multiwindow",
+		show_buffer_close_icons = true,
+		separator_style = "thin",
+		mappings = true,
+		always_show_bufferline = true,
+	},
+	highlights = {
+		fill = {
+			guifg = colors.grey_fg,
+			guibg = colors.black2,
+		},
+		background = {
+			guifg = colors.grey_fg,
+			guibg = colors.black2,
+		},
+		-- buffers
+		buffer_visible = {
+			guifg = colors.light_grey,
+			guibg = colors.black2,
+		},
+		buffer_selected = {
+			guifg = colors.white,
+			guibg = colors.black,
+			gui = "bold",
+		},
+		-- tabs
+		tab = {
+			guifg = colors.light_grey,
+			guibg = colors.one_bg3,
+		},
+		tab_selected = {
+			guifg = colors.black2,
+			guibg = colors.nord_blue,
+		},
+		tab_close = {
+			guifg = colors.red,
+			guibg = colors.black,
+		},
+		indicator_selected = {
+			guifg = colors.black,
+			guibg = colors.black,
+		},
+		-- separators
+		separator = {
+			guifg = colors.black2,
+			guibg = colors.black2,
+		},
+		separator_visible = {
+			guifg = colors.black2,
+			guibg = colors.black2,
+		},
+		separator_selected = {
+			guifg = colors.black2,
+			guibg = colors.black2,
+		},
+		-- modified
+		modified = {
+			guifg = colors.red,
+			guibg = colors.black2,
+		},
+		modified_visible = {
+			guifg = colors.red,
+			guibg = colors.black2,
+		},
+		modified_selected = {
```

Comment:
Hmmmmm, it looks like the stylua config has a super duper big indentation.

Maybe it should be smaller?

Reply:
I think the big indentation is from github. It looks good to me. I shall rename it to .stylua.toml then